src/stravalib/model.py

```diff
@@ -991,8 +996,8 @@ class TimedZoneDistribution(strava_model.TimedZoneRange):
     """
 
     # Type overrides to support pace values returned as ints
-    min: Optional[int | float] = None  # type: ignore
-    max: Optional[int | float] = None  # type: ignore
+    min: Optional[Sequence[int | float]] = None
```

Comment:
I tried to use sequence here as well.... but i am not sure that Sequence supports the types within the  list / tuple / etc. 

Options:
* either we keep the ignore statements 
* there is some other way to do this correctly?